.github/workflows/00_github_staging-validate.yml

```
@@ -0,0 +1,43 @@
+name: Validate Staging Modules
+
+on:
+	pull_request:
```

actionlint (1.7.7)

could not parse as YAML: yaml: line 4: found character that cannot start any token

(syntax-check)